cub/cub/device/dispatch/dispatch_select_if.cuh

```diff
@@ -207,13 +214,14 @@ __launch_bounds__(int(
 {
   using VsmemHelperT = cub::detail::vsmem_helper_default_fallback_policy_t<
     typename ChainedPolicyT::ActivePolicy::SelectIfPolicyT,
-    detail::agent_select_if_wrapper_t<KEEP_REJECTS>::template agent_t,
+    detail::agent_select_if_wrapper_t<KEEP_REJECTS, MayAlias>::template agent_t,
     InputIteratorT,
     FlagsInputIteratorT,
     SelectedOutputIteratorT,
     SelectOpT,
     EqualityOpT,
-    OffsetT>;
+    OffsetT,
+    ScanTileStateT>;
```

Comment:
suggestion: this seems like leftover from the attempt to make memory order part of the tile state template. Given that agent still expects `TileState<OffsetT>`, it seems better not to generalize tile state. This will allow us to catch host-device typing mismatches, if we, for instance, accidentally pass `TileState<int>`.